felix/proto/felixbackend.proto

```diff
@@ -573,6 +573,7 @@ enum RouteType {
   LOCAL_HOST = 4;
   REMOTE_TUNNEL = 5;
   LOCAL_TUNNEL = 6;
+  BLACK_HOLE = 7;
```

Comment:
This is a little unfortunate as itis local to the bpf ep manager :man_shrugging: